### DISCUSSION TITLE
Can not find topic for HV battery temperature max and min

### DISCUSSION THREAD

**Discussion by Gus80** [tillsteinbach/WeConnect-mqtt#306] [General]
Looking for mqtt topic for "TemperatureBatteryStatus" but can not find it :(

**Comment by tillsteinbach** [comment]
Which SW Version does your Car have?

**Comment by Gus80** [comment]
The car has v3.2. (ID 4 GTX)
Using weconnect_mqtt version 0.48.3

**Reply by Gus80** [reply]
Ah think I may have found it...using wrong weconnect_mqtt version in my docker image
Bumped version from 0.46 to 0.48.3

**Comment by Gus80** [comment]
Used wrong version
